python/cloudtik/runtime/spark/conf/spark/spark-defaults.conf

```diff
@@ -35,3 +35,4 @@ spark.sql.warehouse.dir {%spark.sql.warehouse.dir%}
 {%spark.hadoop.hive.metastore.uris%}
 {%spark.sql.hive.metastore.version%}
 {%spark.sql.hive.metastore.jars%}
+{%spark_web_identity_envs%}
```

Comment:
The spark-defaults.conf is shared by all providers.
So for other providers like azure, gcp, the {%spark_web_identity_envs%} string will be left in the Spark configuration.
May cause problems.

Since this is shared placeholder, we can call it {%spark_runtime_envs%} and make sure every case will either set it or clear it.